include/mbedtls/mbedtls_config.h

```diff
@@ -2535,7 +2552,7 @@
  *          library/ssl*_server.c
  *          library/x509.c
  *
- * Requires: MBEDTLS_RSA_C or MBEDTLS_ECP_C
+ * Requires: MBEDTLS_MD_C, MBEDTLS_RSA_C or MBEDTLS_ECP_C
```

Comment:
Note: like with PKCS#1 v1.5, `md.c` is only use to get a size from an alg identifier, so that dependency could easily be avoided if desired. (Again, no action required.)